### ISSUE TITLE
installing out of / breaks the pad.

### ISSUE THREAD

**Issue by tshirtman** [LaKing/ep_codepad#6]
i installed etherpad on /pad on my server, and when i activate codepad, and go to any pad i get
```
'ErrorId: NuaTUwJyOvYziN7grfxg
URL: http://example.com/pad/p/test
UserAgent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:36.0) Gecko/20100101 Firefox/36.0
ReferenceError: getCookie is not defined in http://example.com/codepad/static/js/require-kernel.js line 226 > Function at line 214'
```

when looking at the network log in firefox, i see that highlight.codepad.js and font-awesome.min.css are taken from /static/plugins/ep_codepad instead of /pad/static/plugins/ep_codepad, i suspect an import is done using root of the server, instead of relatively to the current page, it would be nice to fix that.

Thanks a lot :).

**Comment by LaKing** [comment]
I don't really understand that. The proper syntax for the URL / URI is:

https://example.com/p/test.ext

Why do you have pad between the URL and the URI?

**Comment by LaKing** [comment]
I see. ...

Will check your fix asap. Thanks.

**Comment by samveen** [comment]
I've added some more changes and created a pull request against the fork by @tshirtman . In case you merge this before he adds in those commits, please do take a look at the additional changes I made.

**Comment by samveen** [comment]
This is another failure point, but I couldn't think of an elegent non-hack solution for this:

    grep -n "['\"]/files" templates/menuLeftScripts.ejs
    6:            window.location = window.location.protocol + "//" + window.location.host + "/files#" + window.location.pathname + window.location.search;

**Comment by LaKing** [comment]
Theoretically fixed in 1.0.8, but I didnt test it.

There is an optional parameter for settings.json, that solves the issue in an elegant way.
